CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 - Implemented method ParticleSet::resize().
 - Implemented evaluation of a multivariate Gaussian probability density function in method utils::multivariate_gaussian_density.
 - Implemented evaluation of the logarithm of a multivariate Gaussian probability density function in method utils::multivariate_gaussian_log_density.
+- Implemented evaluation of a multivariate Gaussian probability density function for high dimensional inputs in method utils::multivariate_gaussian_density_high_dim.
```

Comment:
Please use `` delimiters to function, classes and everything related to the library. In this case it should be `utils::multivariate_gaussian_density_high_dim`. We'll update the rest of the document prior to the next release.

Reply:
Ok :+1: